inc/functions/minify.php

```diff
@@ -22,9 +22,12 @@ function rocket_minify_files( $buffer, $extension ) {
 	}
 
 	if ( 'js' === $extension ) {
+		$js_files_in_head = '';
 		$header_files     = array();
 		$concatenate      = get_rocket_option( 'minify_concatenate_js', false ) ? true : false;
-		$js_files_in_head = implode( '|', $rocket_js_enqueued_in_head );
+		if ( ! empty( $rocket_js_enqueued_in_head ) ) {
```

Comment:
`implode()` doesn't throw a warning if the 2nd argument is empty, but it does if it's not an array.
I'd rather do that:
```php
if ( $rocket_js_enqueued_in_head && is_array( $rocket_js_enqueued_in_head ) ) {
```